src/src/Helpers/Networking.cpp

```diff
@@ -1544,7 +1544,26 @@ int http_authenticate(const String& logIdentifier,
     event += '=';
     event += httpCode;
     eventQueue.addMove(std::move(event));
+    #if FEATURE_THINGSPEAK_EVENT
+      // Generate event with the response of a 
+      // one value one channel tingspeak request
+      // e.g. (sendToHTTP,api.thingspeak.com,80,/channels/143789/fields/5/last)
+      // where first eventvalue is the channel number, the second the field number
+      // and the third is the value received by the request
+      if (equals(host, F("api.thingspeak.com")) && equals(uri.substring(uri.length() - 5, uri.length()), F("/last")) && httpCode == 200) {
```

Comment:
If I'm not mistaken, the Arduino String class does have something like `endsWith`
So you could then use `&& uri.endsWith(F("/last")`.

Also a check for `httpCode == 200` is less resource intensive, so please check this first.

Reply:
will do...